tests/fixtures/variables.py

```diff
@@ -2,7 +2,7 @@
 from openfisca_core.variables import Variable
 
 
-class TestVariable(Variable):
+class MyVariable(Variable):
```

Comment:
I understand we have a naming conflict with the `pytest` API. However, these new names are not satisfactory.
What about `Stub` or `Fixture`? 🙂